docs/src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -32,6 +32,17 @@ import STYLES from './MarkdownPage.scss';
 
 const getClassName = cssModules(STYLES);
 
+const SEOElements = (props: { description: ?string, title: ?string }) => {
+  const { description, title } = props;
+  return (
+    <Helmet>
+      {title && <title>{title}</title>}
```

Comment:
Helmet has a dedicated prop for `title` as it can do some magic with the title template.

```suggestion
    <Helmet title={title}>
```

Reply:
Could you be more specific?

Reply:
Helmet's docs imply this is the best way to do it

<img width="798" alt="Screen Shot 2020-12-10 at 08 56 20" src="https://user-images.githubusercontent.com/73652/101749383-95230280-3ac5-11eb-9b64-a4fb64eff4e0.png">